java/libs.javacapi/external/binaries-list

```diff
@@ -14,5 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-D9F90625B422470E7F827D5E3C770879B247A5CB com.dukescript.nbjavac:nb-javac:jdk-20+34:api
-40ABE61A0214D3AD8CC7554CB895BF6E9A392EF4 com.dukescript.nbjavac:nb-javac:jdk-20+34
+2D8ACD8AAD9C74FDAAA9D7BBAF4278A57E2D86E0 https://repo1.maven.org/maven2/com/dukescript/nbjavac/nb-javac/jdk-21+26/nb-javac-jdk-21+26-api.jar nb-javac-jdk-21+26-api.jar
+9A9BAB9DC8497962967345A59CDD8CBC9D73A40D https://repo1.maven.org/maven2/com/dukescript/nbjavac/nb-javac/jdk-21+26/nb-javac-jdk-21+26.jar nb-javac-jdk-21+26.jar
```

Comment:
The URL based download should be a fallback, please use our normal maven syntax for artifacts from maven central:

```suggestion
2D8ACD8AAD9C74FDAAA9D7BBAF4278A57E2D86E0 com.dukescript.nbjavac:nb-javac:jdk-21+26:api
9A9BAB9DC8497962967345A59CDD8CBC9D73A40D com.dukescript.nbjavac:nb-javac:jdk-21+26
```